src/app/modify/modify.component.html

```diff
@@ -1 +1,2 @@
 <dsp-resource-view *ngIf="!loading" [iri]="resourceIri"></dsp-resource-view>
+<dsp-progress-indicator *ngIf="loading"></dsp-progress-indicator>
```

Comment:
@kilchenmann this is kinda what I was talking about in the meeting. Is it okay to add this here?